packages/back-end/src/services/vercel.ts

```diff
@@ -68,6 +68,7 @@ export async function createOrgGbKeys(
       organization: orgId,
       secret: false,
       environment: envMapEntry.gb,
+      encryptSDK: false,
```

Comment:
Do we want to support enabling encrypted SDK endpoints via Vercel? Probably not, atleast, initially I'm guessing.